freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/user/UsersyncPoller.java

```diff
@@ -45,31 +50,28 @@
     @Inject
     private UmsEventGenerationIdsProvider umsEventGenerationIdsProvider;
 
-    @Value("${freeipa.syncoperation.poller.enabled:true}")
-    private boolean enabled;
-
     @Inject
     private EntitlementService entitlementService;
 
-    @Scheduled(fixedDelayString = "${freeipa.syncoperation.poller.fixed-delay-millis:60000}",
-            initialDelayString = "${freeipa.syncoperation.poller.initial-delay-millis:300000}")
+    @Scheduled(fixedDelayString = "${freeipa.usersync.poller.fixed-delay-millis:60000}",
+            initialDelayString = "${freeipa.usersync.poller.initial-delay-millis:300000}")
     public void pollUms() {
         try {
-            if (enabled) {
-                syncFreeIpaStacks();
-            }
+            LOGGER.debug("Polling for stale stacks");
+            syncFreeIpaStacks();
         } catch (Exception e) {
             LOGGER.error("Failed to automatically sync users to FreeIPA stacks", e);
         }
     }
 
     @VisibleForTesting
     void syncFreeIpaStacks() {
-        ThreadBasedUserCrnProvider.setUserCrn(INTERNAL_ACTOR_CRN);
-        Optional<String> requestId = Optional.of(UUID.randomUUID().toString());
-        LOGGER.debug("Setting request id = {} for this poll", requestId);
-        MDCBuilder.addRequestId(requestId.get());
         try {
+            ThreadBasedUserCrnProvider.setUserCrn(INTERNAL_ACTOR_CRN);
+            Optional<String> requestId = Optional.of(UUID.randomUUID().toString());
+            MDCBuilder.addRequestId(requestId.get());
+            LOGGER.debug("Setting request id = {} for this poll", requestId);
+
             LOGGER.debug("Attempting to sync users to FreeIPA stacks");
             List<Stack> stackList = stackService.findAllRunning();
```

Comment:
this will list all not terminated stack instead of running. I think you should add a new method which filters for state `AVAILABLE`

Reply:
added.